src/MANYBODY/pair_sw.cpp

```diff
@@ -292,8 +292,22 @@ void PairSW::read_file(char *file)
   // open file on proc 0
 
   if (comm->me == 0) {
-    PotentialFileReader reader(lmp, file, "sw", unit_convert_flag);
-    char *line;
+    std::string potential_name;
```

Comment:
In this case I think the "variant" flag would be overkill unless there is also a requirement to parse the potential files differently, and/or a check whether the potential file would be compatible with a specific SW variant.
From what I read in this PR, the potential files seem to be compatible with either pair style variant, just the actual potential function that is applied is slightly different.

Reply:
The "variant" feature is removed.

Reply:
thanks,